scripts/src/autoInviteStudents/students.json

```
@@ -0,0 +1,18 @@
+[
+  {
+    "firstName": "Павел",
+    "lastName": "Чепиго",
+    "group": "P33211",
+    "nickname": "Tomatae",
+    "year": 2023,
+    "season": "Осень"

```

```suggestion
    "season": "fall"

```

обратите внимание что ваши репозитории имеют префикс `fall`. Предполагается, что этот префикс будет браться как раз из этого поля

---

Действительно это уберёт лишний шаг при составлении названия репозитория. Применил, хотя пока само название не собирал.

Собирать тогда что-то вроде
for (const labName of labNames) {
    reponame = `${student.year}-${student.season}-lab-${labName}-${transliterate(student.lastName)}`
 }
 ?

---

@Tomatae , кажется вы пошли уже править скрипты, здорово! Но оставьте работу другим!
Тут я оставил этот комментарий только чтобы переименовать осень в fall

---

@stepancar В итоге стоит идти в сторону подобных изменений сейчас?